src/trialsynth/base/fetch.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import requests
+
+import gzip
+
+from .config import Config
+from .models import Trial
+import pickle
+logger = logging.getLogger(__name__)
```

Comment:
We can reorder imports as (1) built-in (2) external non-built-in (3) this package, and add a line before making a logger